### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by bossmonkey98**
Cart Page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -13,7 +14,7 @@
 
 h1{
     color: var(--primary-color);
-    font-size: 3rem;
+    font-size: 3rem !important;
```
Comment (t007rushi) [inline style.css:17]: why? important

File: style.css [hunk 2]
```diff
@@ -13,7 +14,7 @@
 
 h1{
```
Comment (t007rushi) [inline style.css:15]: use class selector

File: style.css [hunk 3]
```diff
@@ -180,6 +181,7 @@ i{
 
 .card-badge{
     background-color: var(--primary-color);
+    color: aliceblue;
```
Comment (t007rushi) [inline style.css:184]: use color variables

File: style.css [hunk 4]
```diff
@@ -180,6 +181,7 @@ i{
...
     top: 15px;
     left: 15px;
```
Comment (t007rushi) [inline style.css:186]: use re instead

File: components/pages/cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    width: 2rem;
+    height: 2rem;
+    font-weight: 900;
+    cursor: pointer;
+}
+
+.bill{
+    display: flex;
+    flex-direction: column;
+    margin-top: 8rem !important;
```
Comment (t007rushi) [inline components/pages/cart/cart.css:68]: don't use V.V.IMP change requires

**Review by t007rushi — APPROVED** [review]
DO this mandatory changes

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean. Everything looks good to me. You can merge it.

File: components/pages/cart/cart.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    color: black;
+    background-color: var(--secondary-color);
+    border-radius: 10px;
+}
+
+.item-img{
+    max-width: 20rem;
+    width:100%;
+
+}
```
Comment (Shubhamkr2610) [inline components/pages/cart/cart.css:33]: Suggestion: remove extra spacing

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean . Everything looks good to me. You are good to go

**Pull request merged**